modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,217 @@
+||  ec2.ap-southeast-1.amazonaws.com |  opt-in-not-required  |  ap-southeast-1 ||
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+----
++
+. Install `moactl`, the {product-title} command line utility.
+.. Download the link:https://github.com/openshift/moactl/releases/latest[latest release of moactl] and add it to your path.
+.. Verify your installation by running the following command:
++
+[source, bash]
+----
+$ moactl
+
+Command line tool for MOA.
+
+Usage:
+  moactl [command]
+
+Available Commands:
+  completion  Generates bash completion scripts
+  create      Create a resource from stdin
+  delete      Delete a specific resource
+  describe    Show details of a specific resource
+  edit        Edit a specific resource
+  help        Help about any command
+  init        Applies templates to support Managed OpenShift on AWS clusters
+  list        List all resources of a specific type
+  login       Log in to your Red Hat account
+  logout      Log out
+  logs        Show logs of a specific resource
+  verify      Verify resources are configured correctly for cluster install
+  version     Prints the version of the tool
+
+Flags:
+      --debug     Enable debug mode.
+  -h, --help      help for moactl
+  -v, --v Level   log level for V logs
+
+Use "moactl [command] --help" for more information about a command.
+----
++
+.. (Optional) You can run `moactl completion` to generate a bash completion file. Add this file to the correct location for your operating system. For example, on a Linux machine run the following command to enable moactl bash completion:
++
+[source, bash]
+----
+$ moactl completion > /etc/bash_completion.d/moactl
+
+source /etc/bash_completion.d/moactl
+----
++
+. Run the following command to verify that your AWS account has the necessary permissions:
++
+[source,bash]
+----
+$ moactl verify permissions
+
+I: Validating SCP policies...
+I: AWS SCP policies ok
+----
++
+.. Verify that your AWS account has the necessary quota to deploy an OpenShift cluster.
++
+[source,bash]
+----
+$ export AWS_DEFAULT_REGION=us-west-2 && moactl verify quota
+
+I: Validating AWS quota...
+E: Insufficient AWS quotas
+E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid
+----
++
+[NOTE]
+====
+Sometimes quota varies by region, which may prevent you from deploying. If you receive any errors, try a different region.
```

Comment:
Is it may, can, or might prevent you from deploying?

It's probably can or might. (Although it probably may, but it sounds like it not only may, but _will_.)